game/graphics/pipelines/opengl.cpp

```diff
@@ -191,6 +191,10 @@ static std::shared_ptr<GfxDisplay> gl_make_display(int width,
       g_gfx_data->debug_gui.m_vsync = false;
       g_gfx_data->vsync_enabled = false;
       glfwSwapInterval(false);
+      if (primary_monitor_video_mode->refreshRate > 120) {
```

Comment:
Should this also be done when the refresh rate is *exactly* 120 Hz (which is common on high-end TVs)? I'd wager playing at 150 FPS on a 120 Hz display is an overall better experience than playing at 60 FPS on a 60 Hz display.

That said, there could also be an option to override this regardless of your refresh rate. Even playing at 150 FPS on a 60 Hz display may be a better experience if you're looking to minimize input lag.

To account for floating-point imprecision, the check should probably be > 119 (if `refreshRate` is a float, that is).

Reply:
The 120 is arbitrary so I'm open to changing this. We can support 100fps the same way I did 150 because its a divisor of 300. Because of that, I think it'd be reasonable to set 150FPS mode for anything > 100, and then we'd set 100FPS mode for anything > 60

The reason I was restrictive is just that it's new and being automatically enabled. If you aren't able to run at a consistent 150fps the game breaks.

Also you can change video modes via the REPL so we could add it as a menu option as well